evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1213,21 +1213,10 @@ def test_decision_tree_data_from_estimator_wrong_type(logit_estimator):
         decision_tree_data_from_estimator(est_logit)
 
 
-def test_decision_tree_data_from_estimator_feature_length(fitted_tree_estimators):
-    est_class, _ = fitted_tree_estimators
-    with pytest.raises(ValueError, match="Length mismatch: Expected features has length {} but got list with length {}"
-                                         .format(est_class._component_obj.n_features_, 3)):
-        decision_tree_data_from_estimator(est_class, feature_names=["First", "Second", "Third"])
-
-    features_array = tuple([f'Testing_{col_}' for col_ in range(est_class._component_obj.n_features_)])
-    formatted_ = decision_tree_data_from_estimator(est_class, feature_names=features_array)
-    assert isinstance(formatted_, OrderedDict)
-
-
 def test_decision_tree_data_from_estimator(fitted_tree_estimators):
     est_class, est_reg = fitted_tree_estimators
 
-    formatted_ = decision_tree_data_from_estimator(est_reg, feature_names=[f'Testing_{col_}' for col_ in range(est_reg._component_obj.n_features_)])
+    formatted_ = decision_tree_data_from_estimator(est_reg)
```

Comment:
Removing feature names parameter. Since this test already checks that the output has the feature names but now we're just grabbing them from the estimator rather than setting explicitly, no other code changes needed.